index.html

```diff
@@ -5132,6 +5132,9 @@ <h2>The <code>application/credential+ld+json</code> Media Type</h2>
     the use of a specific media type.
   </p>
 
+  <p>
+    This media type can be used with credentials secured using an <dfn>external proof</dfn>.
+  </p>
```

Comment:
```suggestion
  </p>
  <p>If a [[json-ld]] context is present and indicated by the presence of `+ld` in a media type, the credential MUST be valid <a href="https://www.w3.org/TR/json-ld11/#dfn-json-ld-document">JSON-LD document</a>.</p>
```

Reply:
might not be the right place for this..

Reply:
Seems like the correct place per: https://www.w3.org/TR/json-ld11/#interpreting-json-as-json-ld

<img width="817" alt="Screen Shot 2023-02-11 at 10 08 36 AM" src="https://user-images.githubusercontent.com/8295856/218268441-1e3bb696-8c7a-49e6-b5c2-2f7455a9b7bd.png">

Reply:
I wonder if we need to mirror the guidance on the link header here as well.

Reply:
Three things:

1. We should say "by the presence of `+ld+json`", because there is no such thing as a `+ld` anything, IIRC.
2. We haven't registered [a structured suffix](https://www.iana.org/assignments/media-type-structured-suffix/media-type-structured-suffix.xhtml) for `+ld+json` yet... I don't think that's an issue, but the IETF structured suffix lawyers (or like minded individuals) might grumble.
3. Someone will insist that we test the `MUST` statement (as we do for all of our testable MUST statements), or insist that to be conformant that implementations MUST perform JSON-LD processing... I don't think we want either of those things, so suggest something like "The credential will pass a JSON-LD document conformance checker." (we need better text than that... the point being not to invoke the "How are you going to test this in the test suite!? Does this mean that implementations must perform JSON-LD conformance checking!?" line of questioning).

Don't know if we really need this sort of language... but I'm not opposed to it being somewhere in the spec.

Reply:
Pulling together @mprorock's initial insert, with the comments by @OR13 and @msporny, plus a bit of my own special sauce:
```suggestion
  </p>
  <p>If a [[JSON-LD]] context is present in the body of the document, and indicated by the presence of `ld+json` in the media type, the credential MUST be a valid <a href="https://www.w3.org/TR/json-ld11/#dfn-json-ld-document">JSON-LD document</a>.</p>
```

Reply:
I would like to commit this suggestion, are there any objections to doing so?

Reply:
No objections in 5 days, though I'm concerned about the number of people that have /not/ reviewed this PR.

Reply:
@msporny — (Re-)request more reviews?